doc/release-notes.md

```diff
@@ -35,77 +35,330 @@ the database if you decide to use any pre-0.12.2.2 version.
+If a node connects to the wrong network, it will immediately be disconnected.
+
+New format of network messsage signatures
+-----------------------------------------
+
+We introduce new signature format for Dash-specific network messages,
+read more [here](https://github.com/dashpay/dash/pull/1936) and [here](https://github.com/dashpay/dash/pull/1937).
+We also introduce new spork `SPORK_6_NEW_SIGS` which is going to be used to activate new format after network finishes the upgrade.
+Note that old pre-12.3 nodes won't be able to recognize and verify new signatures after `SPORK_6_NEW_SIGS` activates.
 
-This release also fixes a few crashes and compatibility issues.
+Governance system improvements
+------------------------------
 
+We are no longer using watchdogs already, instead we are including all required information about sentinel
+into masternode pings. With this update we add some additional information and cover everything with a signature
+to ensure that pings were not maleated by some intermediary node. All messages and logic related to watchdogs
+are completely removed now. We also improved proposal message format, as well as proposal validation and processing,
+which should lower network traffic and CPU usage. Handling of triggers was also improved slightly.
+`SPORK_13_OLD_SUPERBLOCK_FLAG` was not used already too and we are completely removing it now.
+
+PrivateSend improvements
+------------------------
+
+PrivateSend collateral is no longer required to be N times of the PrivateSend fee, instead any input
+which is in the [1..N] range can be used as a collateral. Inputs that are in the [1..2) range will now
+be used in collaterals with OP_RETURN outputs which will allow such inputs to be completely consumed
+(previousely they were not used at all and instead were kept in wallet and global UTXO forever).
+There are also some minor fixes which should slightly improve mixing process too.
+
+Additional indexes cover P2PK now
+---------------------------------
+
+Additional indexes like `addressindex` etc. process P2PK outputs correctly now. Note, that these indexes will
+not be re-built automatically on wallet update, you must reindex manually to update indexes with P2PK outputs.
+
+Support for pruned nodes in Lite Mode
+-------------------------------------
+
+It is now possible to run a pruned node which stores only some recent blocks and not the whole blockchain.
+However this option is only available in so called Lite Mode. In this mode no Dash specific feature are available
+for such node which means that node will NOT be able to fully validate blockchain (masternode payments and superblocks).
+PrivateSend and InstantSend functions are also disabled on such node. You can think of such node as of an SPV-like one
+in terms of security and validation - it relies a lot on surrounding nodes, so please keep this in mind if you decide to
+use it for something.
+
+RPC changes
```

Comment:
Not sure how comprehensive this is intended to be. Some other RPC items:
- `getrawmempool`'s added InstantSend-related info
- `getgovernanceinfo` changed from mnwatchdog->sentinelping
- `getgenerate`/`setgenerate` removed
- `getaddednodeinfo` - dummy parameter removed
- `getwalletinfo` - add PS balance

Reply:
```
- `getgenerate`/`setgenerate` removed
- `getaddednodeinfo` - dummy parameter removed
```
These two were changed via backports I believe, so shouldn't be in that list imo, but I'll add the rest of the list. Thanks! 👍